### ISSUE TITLE
Rosetta crashes for custom user models without is_superuser and groups

### ISSUE THREAD

**Issue by EmilStenstrom** [mbi/django-rosetta#131]
Hi,
I have replaces the default User model with one that doesn't have is_superuser and groups at all. This means the the permission check when starting rosetta crashes since it expects either of those properties.

``` python
@property
def is_superuser(self):
    return True
```

Suggestion: 
- If is_superuser doesn't exist, treat it as False
- If groups doesn't exist, let me specify a list of user identifiers in settings instead. To know what field to look up the file against you can look at CustomUser.USERNAME_FIELD which is required for custom user models.

Does this make sense?

**Comment by mbi** [comment]
Hmm, haven't actually looked at the code, but have you tried playing with `ROSETTA_ACCESS_CONTROL_FUNCTION`? See https://django-rosetta.readthedocs.org/en/latest/settings.html

**Comment by EmilStenstrom** [comment]
Hi,
Thanks for a quick reply. This works great for us:

``` python
ROSETTA_TRANSLATORS = ["mail@example.com", ...]
ROSETTA_ACCESS_CONTROL_FUNCTION = "settings.has_rosetta_access"
def has_rosetta_access(user):
    is_translator = (user.email in ROSETTA_TRANSLATORS)
    return user.is_authenticated() and (user.is_staff or is_translator)
```

Maybe add some error handling (an exception?) that points users to ROSETTA_ACCESS_CONTROL_FUNCTION when is_superuser and groups is missing?

**Comment by mbi** [comment]
This should be fixed in e117f1c135a457385ff9dd7fc813afd539684c07
It's hard to test this, could you please unset `ROSETTA_ACCESS_CONTROL_FUNCTION` and confirm you get an exception?

Thanks.

**Comment by Ligadata-YazeedHasan** [comment]
Hi @EmilStenstrom 
where should I add that part of the code? not sure if the settings file has any rule to play here
